apiary.apib

```diff
@@ -2058,7 +2058,8 @@ Get details for a single invoice.
                     + payment (object)
                         + Include Money
             + payment_reference: `+++084/2613/66074+++` (string)
-            + note: `'Some extra remarks about the invoice'` (string)
+            + note: `Some extra remarks about the invoice` (string)
+            + expected_payment_method (ExpectedPaymentMethod, optional)
```

Comment:
Q: Do we return optional properties in a .info endpoint. I think this needs to be nullable instead of optional